tests/console/nvme_checks.pm

```diff
@@ -0,0 +1,125 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Smoke tests for NVM Express.
+#
+# - Install nvme-cli which is a tool to manage NVM Express
+# - Check sys path that should be hold details
+# - Check if we have any NVMe controllers on the PCI bus
+# - Check if we have NVMe devices
+# - Check if we have namespace
+# - Send Identify Admin Command to the NVMe controller
+# - Issue a Read command both using the helper and manually
+# - Compare it to what we have with a regular dd
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base "installbasetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use scheduler 'get_test_suite_data';
+
+sub _check_basic_installation {
+    my $nvm_test_data = shift;
+    assert_script_run("lspci -nn | grep -i nvm");
+    assert_script_run("test -d $nvm_test_data->{nvme_sys_path} || echo nvme directory not found in $nvm_test_data->{nvme_sys_path}", fail_message => "$nvm_test_data->{nvme_sys_path} not found");
+    assert_script_run("test -c /dev/$nvm_test_data->{nvm_char_device} || echo 'nvm character devices not found/exist'", fail_message => "/dev/$nvm_test_data->{nvme_char_device} not found");
```

Comment:
Do we need `echo` when we already have `fail_message`? I don't think we need both. On top `echo` might hide result `test`, as we use logical OR `||`

Reply:
Kinda expected this comment. `or` removed

Reply:
Please, could you wait for resolving the conversations, I almost miss this one.
Could you please check the remaining echos in the files?

Reply:
Resolving as it has been mentioned later below.